### PR TITLE
feature/BOUND-33-Update-Display-Board-General

### PR DESCRIPTION
**Pull request by SCustodero**

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -336,7 +336,7 @@ def reveal_cards(self, marshall_current_idx, repeat):
       for i in range(repeat):
         self.cards_in_play[marshall_current_idx + i].revealed = True
     
-    def display_board_general(self):
+    def display_board_general(self, player = 'marshall'):
```
Comment (EthanThomas0523) [inline source/CardGames.py:351]: Nice job, Sean! I like the way you did your function.

Comment (georgiok741) [inline source/CardGames.py:351]: Nice touch here!

Comment (SCustodero) [reply source/CardGames.py:351]: Thank you sir

File: source/CardGames.py [hunk 2]
```diff
@@ -346,7 +346,10 @@ def display_board_general(self):
               card = stack[0]
             else:
               card = stack
-            image = card.image[idx] if card.revealed else card.cardBack[idx]
+            if player == 'fugitive' or card.revealed:
```
Comment (youraveragedeveloper7) [inline source/CardGames.py:362]: Looks great here! I'm assuming that when we call the function, we need to explicitly set the player parameter as fugitive?

**Review by youraveragedeveloper7 — APPROVED** [review]

File: source/CardGames.py (hunk 2 above)
Comment (SCustodero) [reply source/CardGames.py:362]: Thanks! Yes, if its the fugitive's turn then it has to be passed exactly as 'fugitive'

**Review by EthanThomas0523 — APPROVED** [review]

**Review by georgiok741 — APPROVED** [review]

File: source/CardGames.py (hunk 1 above)
Comment (SCustodero) [reply source/CardGames.py:351]: Thanks brother

**Pull request merged**